samples/CommunityToolkit.Maui.Markup.Sample/Pages/SettingsPage.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using CommunityToolkit.Maui.Behaviors;
+using CommunityToolkit.Maui.Markup.Sample.Constants;
+using CommunityToolkit.Maui.Markup.Sample.Pages.Base;
+using CommunityToolkit.Maui.Markup.Sample.Services;
+using CommunityToolkit.Maui.Markup.Sample.ViewModels;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Layouts;
+
+namespace CommunityToolkit.Maui.Markup.Sample.Pages;
+
+class SettingsPage : BaseContentPage<SettingsViewModel>
+{
+	public SettingsPage(SettingsViewModel settingsViewModel) : base(settingsViewModel, "Settings")
+	{
+		Content = new AbsoluteLayout
+		{
+			Children =
+			{
+				new Label { Text = "Top Stories To Fetch", TextColor = ColorConstants.PrimaryTextColor, HorizontalTextAlignment = TextAlignment.Center }
+					.LayoutFlags(AbsoluteLayoutFlags.XProportional | AbsoluteLayoutFlags.WidthProportional)
+					.LayoutBounds(0,0,1,40),
+
+				new Entry { Keyboard = Keyboard.Numeric, BackgroundColor = Colors.White }
+					.LayoutFlags(AbsoluteLayoutFlags.XProportional | AbsoluteLayoutFlags.WidthProportional)
+					.LayoutBounds(0.5,45,0.8,40)
+					.Behaviors(new NumericValidationBehavior
+					{
+						MinimumValue = SettingsService.MinimumStoriesToFetch,
+						MaximumValue = SettingsService.MaximumStoriesToFetch,
+						Flags = ValidationFlags.ValidateOnValueChanged,
+						InvalidStyle = new Style<Entry>((Entry.TextColorProperty, Colors.Red)),
+						ValidStyle = new Style<Entry>((Entry.TextColorProperty, ColorConstants.PrimaryTextColor)),
+					})
+					.Bind(Entry.TextProperty, nameof(SettingsViewModel.NumberOfTopStoriesToFetch)),
+
+				new Label
+				{
+					Text = $"Top Stories To Fetch", TextColor = ColorConstants.PrimaryTextColor, HorizontalTextAlignment = TextAlignment.Center,
+					FontAttributes = FontAttributes.Italic, FontSize = 12
+				}
+					.Bind<Label, int, int, string>(
+						Label.TextProperty,
+						binding1: new Binding { Source = SettingsService.MinimumStoriesToFetch },
+						binding2: new Binding { Source = SettingsService.MaximumStoriesToFetch },
+						convert: ((int minimum, int maximum) values) => string.Format(CultureInfo.CurrentUICulture, "The number must be between {0} and {1}.", values.minimum, values.maximum))
+					.LayoutFlags(AbsoluteLayoutFlags.XProportional | AbsoluteLayoutFlags.WidthProportional)
+					.LayoutBounds(0,90,1,40),
```

Comment:
Technically the multi-binding isn't needed since it is a `OneTime` binding (ie the binding source is constant).

However, I like keeping the multi binding to demonstrate how to use multi binding it in the sample app!

(Here's what it would look like without the binding)
```cs
				new Label { Text = $"The number must be between SettingsService.MinimumStoriesToFetch and SettingsService.MaximumStoriesToFetch.", TextColor = ColorConstants.PrimaryTextColor }
					.LayoutFlags(AbsoluteLayoutFlags.XProportional | AbsoluteLayoutFlags.WidthProportional)
					.LayoutBounds(0,90,1,40)
					.TextCenter()
					.Font(size: 12, italic: true)
```

Reply:
I've got so much to learn! Thanks for all these suggestions.

Yes I like the idea of showing off a multi binding here despite the value not changing. I did also consider whether we should localise eventually to show that concept off too. Not in scope of this PR though.